### PR TITLE
remove incompatible characters for enum value

### PR DESCRIPTION
**Pull request by eledumorales**
Description
-----------
Please describe your pull request and any relevant information.

Checklist
-----------

- [ ] I have added appropriate tests if this PR fixes a bug or adds a feature.
- [ ] All [status checks](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-status-checks) (tests, linting, etc...) are passing.

### CODE REVIEW

File: lib/hq/graphql/enum.rb [hunk 1]
```diff
@@ -49,7 +49,7 @@ def self.with_model(
       lazy_load do
         records = scope ? klass.instance_exec(&scope) : klass.all
         records.each do |record|
-          value "#{prefix}#{record.send(value_method).delete(" ")}", value: record
+          value "#{prefix}#{record.send(value_method).sub(/^((::)?\w+)::/, "").delete(" ")}", value: record
```
Comment (dnnyjns) [inline lib/hq/graphql/enum.rb:52]: It doesn't look like it removes spaces anymore. What are you trying to remove? Semi-colons? Anything else?

Comment (dnnyjns) [reply lib/hq/graphql/enum.rb:52]: It looks like this is breaking a handful of specs now. Are you running them locally? We don't have github actions setup yet to run them in CI

Comment (eledumorales) [reply lib/hq/graphql/enum.rb:52]: I was having trouble with the colons.. the delete(" ") doesn't remove the spaces?

Comment (dnnyjns) [reply lib/hq/graphql/enum.rb:52]: I didn't see `delete(" ")` still on the end 🤦‍♂️.

Can this just be changed to this?
```suggestion
          value "#{prefix}#{record.send(value_method).delete(": ")}", value: record
```

**Pull request merged**